File: ruoze-homework/src/main/scala/com/ruoze/bigdata/tututuhomework/day20201013/sparketlwithAccumulator/utils/FileUtils.scala
package com.ruoze.bigdata.tututuhomework.day20201013.sparketlwithAccumulator.utils

import org.apache.hadoop.conf.Configuration
import org.apache.hadoop.fs.{FileSystem, Path}

object FileUtils {

  @throws[Exception]
  def delete(conf: Configuration, output: String): Unit = {
    val fileSystem = FileSystem.get(conf)
    val outputPath = new Path(output)
    if (fileSystem.exists(outputPath)) fileSystem.delete(outputPath, true)
  }

}

File: doc/上课笔记/03-MapReduce编程01/资料/FileUtils.java
package com.ruozedata.bigdata.utils;

import org.apache.hadoop.conf.Configuration;
import org.apache.hadoop.fs.FileSystem;
import org.apache.hadoop.fs.Path;

/**
 * @author PK哥
 **/
public class FileUtils {

    public static void deleteOutput(Configuration configuration, String output) throws Exception {
        FileSystem fileSystem = FileSystem.get(configuration);
        Path outputPath = new Path(output);
        if(fileSystem.exists(outputPath)) {
            fileSystem.delete(outputPath, true);
        }
    }
}
